docs/data/data-grid/master-detail/DetailPanelAutoHeight.tsx

```diff
@@ -221,8 +222,10 @@ const rows = [
 type Customer = typeof rows[number];
 
 export default function DetailPanelAutoHeight() {
-  const getDetailPanelContent: DataGridProProps['getDetailPanelContent'] =
```

Comment:
Why removing it?

Reply:
Because TS considered that `row === any` so it complained, which is why I used the `GridRowParams` instead
But it works with `const getDetailPanelContent = React.useCallback<NonNullable<DataGridProProps['getDetailPanelContent']>>(` which is better